stable/cockroachdb/README.md

```diff
@@ -1,5 +1,8 @@
 # CockroachDB Helm Chart
 
+## Documentation
+Below is a brief overview of operating the CockroachDB Helm Chart and some specific implementation details.  For additional information, please see https://www.cockroachlabs.com/docs/v19.1/orchestrate-cockroachdb-with-kubernetes-insecure.html
```

Comment:
Best to use `/docs/stable/orchestrate...` instead of `/docs/v19.1/orchestrate...`. `stable` will always point at the stable version of our product/docs. I think I'd make this a link as follows:

```
please see the [CockroachDB documentation](https://www.cockroachlabs.com/docs/stable/orchestrate-cockroachdb-with-kubernetes-insecure.html).